storage/src/system/protocol_upgrade.rs

```diff
@@ -1009,6 +1012,72 @@ where
         Ok(())
     }
 
+    /// Handle seignorage snapshot migration.
+    pub fn handle_seignorage_snapshot_migration(
```

Comment:
This will have to be refactored once https://github.com/casper-network/casper-node/pull/4897 is merged to handle the auction contract either being migrated to `AddressableEntity` or not.

Reply:
After some refactoring this is probably no longer the case.